viewer/packages/tools/src/Geomap/Geomap.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as GEOTHREE from 'geo-three';
+import { Cognite3DViewer } from '@reveal/core';
+import { MapboxStyle, MapConfig, MapProviders } from './MapConfig';
+import { MapProvider, MapView } from 'geo-three';
+
+export class Geomap {
+  private readonly _viewer: Cognite3DViewer;
+  private _map: MapView;
+
+  constructor(viewer: Cognite3DViewer, mapConfig: MapConfig) {
+    this._viewer = viewer;
+    const mapProvider = this.getMapProvider(mapConfig);
+    this._map = new GEOTHREE.MapView(GEOTHREE.MapView.PLANAR, mapProvider, mapProvider);
+    this._viewer.addObject3D(this._map);
```

Comment:
return directly, no need to store in local var first.

Reply:
`this._map` is needed for `setMapProvider()` & also to position the map according to the Latitude & Longitude values.